subsys/bluetooth/host/monitor.c

```diff
@@ -52,6 +52,9 @@ enum {
 };
 
 static atomic_t flags;
+#if defined(CONFIG_BT_DEBUG_MONITOR_RTT) || !defined(CONFIG_LOG_MODE_MINIMAL)
+static bool panic_mode;
+#endif
```

Comment:
Given the use of it is only for `CONFIG_BT_DEBUG_MONITOR_RTT` then I suggest to move the declaration of the variable to the `#if defined(CONFIG_BT_DEBUG_MONITOR_RTT)` part on line 100.